pages/learn/deploy/release-strategy/release-policy.md

```diff
@@ -7,45 +7,40 @@ excerpt: Manage the fleet and device release policy
 
 When managing a fleet, you may require devices to be running different releases. By default, fleets follow a rolling release policy where new releases are deployed to all devices in the fleet once successfully built. However, you can customize this behavior, so the fleet or individual devices remain on a fixed release, by utilizing release pinning.
 
-You may define the fleet and device release policies via the [{{ $names.cloud.lower }} dashboard]({{ $links.dashboardUrl }}) or programmatically through the [API][api] or [SDKs][sdk].
-
-__Note:__ It is currently not possible to specify the fleet or device release policy via the {{ $names.cli.lower }}.
+You may define the fleet and device release policies via the [{{ $names.cloud.lower }} dashboard]({{ $links.dashboardUrl }}) or programmatically through the [API][api], [SDKs][sdk], or [CLI][cli].
 
 ## Pin fleet to a release
 
-In the fleets dashboard, there is a _Release policy_ header. By default, this policy is set to `track latest`, which means that new releases are immediately deployed to all devices in the fleet when built. Expanding this _Release policy_ dropdown menu displays all releases for the fleet, and you can select a specific release to pin the fleet to.  When the fleet is pinned, all devices are updated to run the pinned release, and in the future, releases will not be deployed until the _Release policy_ is updated to a newer release or set to `track latest`.
+You can set the release the fleet is pinned to either via the Releases card on the On the Fleet summary page or the `Pin to release` setting on the Fleet settings page. By default, the setting is set to `track latest`, which means that new releases are immediately deployed to all devices in the fleet when built. The dropdown menu at both of the aforementioned locations displays all releases for the fleet, and you can select a specific release to pin the fleet to.  When the fleet is pinned, all devices that are not currently pinned to a release are updated to run the pinned release, and in the future, releases will not be deployed until the pinned release is updated to a different release or set to `track latest`.
+
+<img src="/img/common/release-policy/releases_card_dropdown.png" width="100%" alt="Pin fleet to release from Releases card">
+<img src="/img/common/release-policy/pin_to_release_setting.png" width="100%" alt="Pin fleet to release setting">
 
-<img src="/img/common/release-policy/mNCaLF8.gif" width="100%" alt="Pin fleet to release">
+You can also pin a fleet to a release via the CLI as follows:
```

Comment:
Can we link the CLI docs as well for this? I remember creating this issue which makes me realize we don't have docs to begin with: https://github.com/balena-io/balena-cli/issues/2604

Reply:
Yes we can do that, but in the meantime these docs are very outdated so I would prefer not to block on that. I think we can update these docs witht he CLI docs in a followup PR